syntax = "proto3";

import "google/protobuf/empty.proto";

option go_package = "github.com/goriiin/kotyari-bots_backend/api/gen/grpc/url_fetcher";

package url_fetcher;

service ProfileService {
  rpc StartFetching(google.protobuf.Empty) returns (google.protobuf.Empty);
}